love/src/components/GeneralPurpose/Azimuth/Azimuth.jsx

```diff
@@ -94,7 +95,7 @@ export default class Limits extends Component {
             />
 
             {/* Gauge */}
-            <circle
+            {/* <circle
               style={{
                 transition: 'transform 1.5s linear 0s',
                 transformBox: 'fill-box',
```

Comment:
This comment block should be removed.

Reply:
In this moment commented, but it can change in the future. Used for the showing representation of an azimuth without green line why it's not have limits with case of alert.